### PR TITLE
Forms

### PR DESCRIPTION
**Pull request by see-why**
- Create forms to perform the following functions:
    - Create a Post on behalf of the current_user.
    - Create a comment on behalf of the current_user.

- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -8,5 +8,22 @@ def show
...
     @post_counter = params[:post_counter]
+    @comment = Comment.new
+  end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    current_user = User.find(params[:user_id])
```
Comment (Lameck1) [inline app/controllers/posts_controller.rb]: - [ ] As of now the current user is the first user just as provided by the application controller. Therefore, there is no need to find the current_user.

  Look at the requirements below:
  
  - > Creates a Post on behalf of the current_user.
  
  Please refactor as necessary.

File: app/controllers/likes_controller.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+class LikesController < ApplicationController
+  def create
+    post = Post.find(params[:post_id])
+    current_user = post.author
```
Comment (Lameck1) [inline app/controllers/likes_controller.rb]: - [ ] Same issue here, as discussed above. The current_user is still the first User in the app. Please use the current user as is.

**Review by Lameck1 — APPROVED** [review]
Hi, @see-why, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Please excuse the mistake above. It's a suggestion of changes, not approval.

**Review by Lameck1 — APPROVED** [review]
Hi, @see-why,

You've done well working on the concerns of the previous review 👍🏼.
The tasks for this milestone are therefore successfully implemented 💯.

Your project is now complete, you are free to merge this branch :shipit:

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/likes_controller.rb (hunk 2 above)
Comment (see-why) [reply app/controllers/likes_controller.rb]: Same tenacity here.
The change has been made here as well.

😄

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (see-why) [reply app/controllers/posts_controller.rb]: Thanks for pointing this out, I got pumped and implemented the new functionalities for every user.

😄

**Pull request merged**